ciao-4.11/contrib/lib/python3.5/site-packages/ciao_contrib/cda/csccli.py

```diff
@@ -818,24 +813,23 @@ def fix_dm_no_g_type( page ):
       cscview returns.  The distinction between D, E, F and G,
       are only on output.  When read in, they are all parsed the
       same (atof or strtod) so we can just change the "(Gnumber)"
-      to be "(Fnumber)"    
+      to be "(Fnumber)"
     """
     import re as re
-    
-    fixed = re.sub( '\(G[0-9\.]*\)',
-                lambda x: x.group(0).replace("G", "F"),
-                page )
-    return fixed
 
+    fixed = re.sub(r'\(G[0-9\.]*\)',
```

Comment:
Needs review.

Since string values are already escaped, the `\` now gets into the regular expression which may foul up the parsing.

Reply:
This works ok here.

There are other similar changes in the `coords` package ... definitely sure we're not hitting all those code paths in the regression tests so not sure if there will be an change in behavior.

Reply:
If you don't add a leading r to the string then tools like `pycodestyle` complain. Whether that's a good or bad thing I don't know.